Dockerfile

```diff
@@ -41,6 +41,7 @@ RUN find /opt/pwn.college/script -type f -exec ln -s {} /usr/bin/ \;
 EXPOSE 22
 EXPOSE 80
 EXPOSE 443
+EXPOSE 5912
```

Comment:
I don't think we need this

Reply:
> The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published. 
This is simply to document that if you want to see the output of the build process, you should forward this port to be able to connect in with VNC.

Reply:
I see. Are we concerned about the build process failing? Obviously this makes sense from a dev perspective, but I think I see these EXPOSEs here as an interface to dojo-consumers. Our black-box dojo speaks on 22/80/443 for users.